cvat/apps/engine/management/commands/export_completed_annotations.py

```diff
@@ -28,7 +28,11 @@ def handle(self, *args, **options):
             options['dump_folder'] = os.path.join("/home/django/share/out/", datetime.now().strftime("%Y_%m_%d"))
         verbose = options['verbose']
         exported_tasks = []
-        for task in Task.objects.filter(status='completed', owner__isnull=False, assignee__isnull=False):
+        for task in Task.objects.filter(status='completed', assignee__isnull=False):
+            if task.owner is None:
+                if verbose:
+                    print('Task ' + task.name + 'does not have an owner, skipping...')
+                continue
```

Comment:
sry...
I misspoke earlier
```suggestion
        for task in Task.objects.filter(status='completed', owner__isnull=False):
            if task.assignee is None:
                if verbose:
                    print('Task ' + task.name + 'does not have an assignee, skipping...')
                continue
```